mbuild/compound.py

```diff
@@ -1298,11 +1298,16 @@ def maxs(self):
         """Return the maximum x, y, z coordinate of any particle in this compound."""
         return self.xyz.max(axis=0)
 
-    def get_boundingbox(self):
+    def get_boundingbox(self, pad_box=False):
```

Comment:
should we have `pad_box` default to `True`?

Reply:
Since it could be possible for a user to incorrectly set their compounds up I think it might be better to catch this and make the user have to rectify it in some way. I think they should toggle the pad box intentionally. But I am amenable to changing this value to true.